### ISSUE TITLE
Suggested date format should be part of the form label element

### ISSUE THREAD

**Issue by jbickar** [SU-SWS/stanford_events_importer#42]
```
<div class="form-item form-type-textfield form-item-field-stanford-event-datetime-value-1-min-date">
<label for="edit-field-stanford-event-datetime-value-1-min-datepicker-popup-1" class="element-invisible">Date </label>
<input type="text" class="form-text" maxlength="30" size="20" value="" name="field_stanford_event_datetime_value_1[min][date]" id="edit-field-stanford-event-datetime-value-1-min-datepicker-popup-1">
<div class="description"> E.g., Oct 21 2014</div>
</div>
```

Ideally (I think), the "E.g., Oct 21 2014" should be in the &lt;label&gt; element

**Comment by jbickar** [comment]
This is on Views exposed filters
